features/FEATURE_BLE/source/generic/FileSecurityDb.cpp

```diff
@@ -265,6 +267,29 @@ void FileSecurityDb::set_entry_peer_sign_counter(
     }
 }
 
+void FileSecurityDb::set_local_csrk(
+    const csrk_t &csrk
+) {
+    this->SecurityDb::set_local_csrk(csrk);
+    db_write(&_local_csrk, DB_OFFSET_LOCAL_CSRK);
+}
+
+void FileSecurityDb::set_local_sign_counter(
+        sign_count_t sign_counter
+) {
+    this->SecurityDb::set_local_sign_counter(sign_counter);
+    db_write(&_local_sign_counter, DB_OFFSET_LOCAL_SIGN_COUNT);
```

Comment:
We talked about it, is it something we want to do everytime the sign counter change ? 
I would also either override it in the KVStore implementation and just call the parent method or leave a note here explaining why it isn't overridden (too many writes...)

Reply:
that is fixed on the other branch, I'll rebase this